lib/bundler/rubygems_integration.rb

```diff
@@ -48,6 +48,23 @@ def loaded_specs(name)
       Gem.loaded_specs[name]
     end
 
+    def setup(specs, opts = {})
```

Comment:
imho this shouldn't be a method on `RubygemIntegration` since it doesn't map to a method in rubygems

Reply:
Sorry, I didn't realize that was a design goal here. There are already quite a few methods on here that don't seem to map 1-1 to a method on Gem, including most of the methods that this calls (e.g. `mark_loaded`, `replace_*`). Is the eventual goal to split those out of here to make this a more pure wrapper around the Gem APIs?